samples/meetings-attendance-report/nodejs/README.md

```diff
@@ -0,0 +1,141 @@
+---
+page_type: sample
+description: This is a sample application which demonstrates how to get meeting attendance report using Graph API and send it in meeting chat.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+- javascript
+extensions:
+contentType: samples
+createdDate: "20-08-2022 00:30:15"
+---
+
+# Bot to show the attendance report of the meeting using Microsoft Graph API.
+
+This is a sample application which demonstrates how to get meeting attendance report using Graph API and send it in meeting chat.
+
+## Key features
+
+When meeting end attendance report card is sent by the bot
```

Comment:
When meeting ends, attendance report card is sent by the bot.

Reply:
Done